src/compile/scale/range.ts

```diff
@@ -220,6 +220,9 @@ export function defaultRange(
         // Only nominal data uses ordinal scale by default
         return type === 'nominal' ? 'category' : 'ordinal';
       }
+      if (isContinuousToContinuous(scaleType)) {
+        return ['#f7fbff', '#0e427f']; // also update interpolation if you change this
```

Comment:
Wonder if it's worth adding to Vega config.

Reply:
@jheer?

Reply:
If we want to be consistent with [the `ramp` / `ordinal` range](https://github.com/vega/vega-parser/blob/master/src/config.js#L205), perhaps we could use

```
{
  scheme: 'blues',
  extent: [0.2, 1],
  count: 2
}
```

Reply:
<img width="494" alt="screen shot 2018-07-17 at 09 16 15" src="https://user-images.githubusercontent.com/589034/42831029-16406718-89a2-11e8-9e2d-8049bf0adb8a.png">

Reply:
Haha, ok.  Bad suggestion then.